tracy-client/src/lib.rs

```diff
@@ -76,22 +76,21 @@ pub mod internal {
         crate::SpanLocation { _internal: () }
     }
 
-    #[inline(always)]
+    #[must_use]
     pub const unsafe fn create_frame_name(name: &'static str) -> crate::frame::FrameName {
         crate::frame::FrameName(name)
     }
 
-    #[inline(always)]
+    #[must_use]
     pub const unsafe fn create_plot(name: &'static str) -> crate::plot::PlotName {
         crate::plot::PlotName(name)
     }
 
-    #[inline(always)]
     /// Safety: `name` must be null-terminated, and a `Client` must be enabled
     pub unsafe fn set_thread_name(name: *const u8) {
         #[cfg(feature = "enable")]
         unsafe {
-            sys::___tracy_set_thread_name(name.cast())
+            sys::___tracy_set_thread_name(name.cast());
```

Comment:
In this case the lack of the semicolon actually serves a good purpose – if an upgrade of the `tracy-client-sys` introduced a return type for this function, it would fail to compile, which is something quite desirable, if only to double check that we can ignore it.

Reply:
Smart! In that case, let's make it explicit.